docs/usage/tutorials.md

```diff
@@ -0,0 +1,563 @@
+
+```bash
+nextflow run nf-core/taxprofiler -r 1.1.0 -profile docker \
+--input samplesheet.csv --databases database.csv --outdir ./results \
+--perform_shortread_qc \
+--perform_shortread_hostremoval --hostremoval_reference GCF_000819615.1_ViralProj14015_genomic.fna.gz \
+--perform_runmerging --save_runmerged_reads \
+--run_centrifuge --run_kaiju --run_kraken2 \
+--run_profile_standardisation \
+--max_cpus 2 --max_memory '6.GB'
+```
+
+:::info
+With all Docker containers pre-downloaded, this run took 2 minutes and 31 seconds on a laptop running Ubuntu 22.04.2 with 32 GB RAM and 16 CPUs.
+If you are running nf-core/taxprofiler for the first time, expect this command to take longer as Nextflow will have to download each software container for each step of the pipeline.
+:::
+
+To break down each line of the command:
+
+- Tell Nextflow to run nf-core/taxprofiler with the particular version and using the Docker container system
+- Specify the input and outputs, i.e., paths to the `samplesheet.csv`, `database.csv`, and directory where to save the results
+- Turn on basic quality control of input reads: adapter clipping, length filtering, etc
+- Turn on the removal of host or contaminant reads, and specify the path to reference genome of this
+- Turn on run merging, i.e., combine the processed input reads of the multiple libraries into each sample, and save these reads (e.g. for downstream use)
+- Turn on the different taxonomic profiling tools you wish to use
+- Turn on profile standardisation and multi-sample taxon tables
+- (Optional) provide a _cap_ to the maximum amount of resources each step/job of the pipeline can use
+
+:::warning
+The `--max_cpu`, `--max_memory`, `--max_time` parameters _do not_ increase the amount of memory a step of the pipeline uses!
+They simply prevent Nextflow requesting more than this threshold, e.g. more than available on your machine.
+To learn how to increase computational resource to the pipeline, see the central [nf-core documentation](https://nf-co.re/docs/usage/configuration).
+:::
+
+The pipeline run can be represented (in a simplified format!) as follows
+
+```mermaid
+graph LR
+0([FASTQs]) --> X[FastQC] --> A[FastP] --> Y[FastQC] --> B[BowTie2]
+0([FASTQs]) --> X[FastQC] --> A[FastP] --> Y[FastQC] --> B[BowTie2]
+
+3([Reference FASTA]) -----> B
+
+2([Databases]) -------> D[Kraken2]
+2([Databases]) -------> E[Centrifuge]
+2([Databases]) -------> E[Centrifuge]
+2([Databases]) -------> F[Kaiju]
+
+B--> C[Run Merging]
+B--> C[Run Merging]
+C --> D[Kraken2]
+C --> E[Centrifuge]
+C --> F[Kaiju]
+
+D --> G[combinekreports]
+E --> H[combinekreports]
+F --> I[Kaiju2Table]
+
+D ---> J[TAXPASTA]
+E ---> J[TAXPASTA]
+F ---> J[TAXPASTA]
+
+X ---> K[MultiQC]
+A ---> K[MultiQC]
+B ---> K[MultiQC]
+D ---> K[MultiQC]
+E ---> K[MultiQC]
+F ---> K[MultiQC]
+```
+
+:::publication{title=""}
```

Comment:
what is the reasoning behind using the publication admonition here? would you like to have a success admonition similar to https://multiqc.info/docs/development/contributing/#success ?

Reply:
Okay, I can add a "success" admonition with a tick-mark. the icons are implemented rather low-level, so not so easy to make  to change them I am afraid, but I will have a look if I could add a fontawesome fallback

Reply:
Another one that would be useful would be a 'tip' category - in fact a green box with light bulb emoji would also work. I don't full understand what the 'publication' one would be used for thinking about it

Reply:
the publications are used here (was a bit of a hack, I confess 🙂) https://nf-co.re/publications

Reply:
hmm, not sure about that. We already have Note and Info, which feel to me very similar to a Tip element (except for the color). But can be convinced if you think there is a big enough difference 🙂

Reply:
I actually don't see the difference between note and info tbh 😅.

Both of those are used for providing context, tip is a bonus extra (if that makes sense)?

Reply:
Done https://nf-co.re/docs/contributing/special_content#tip and done https://nf-co.re/docs/contributing/special_content#custom-icon

Reply:
Thank you @mashehu ! (also loving the example text btw :grimacing: ))